mock banking page/index.html

```diff
@@ -51,6 +51,7 @@ <h2>Borrowing</h2>
                 </tr>
             </table>
         </div>
+        <div id="widget-container"></div>
```

Comment:
why did you move it?

Reply:
include documentation that this is a temporary change

Reply:
Done